src2/app.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const router = express.Router();
+
+export const app = express();
+
+const products = [
+    {
+        name: 'product',
+        reviews: {
+            Nick: 'jkjkjlj',
+            Jan: 'jkjljlkjjkjjkjkj'
+        }
+    },
+    {
+        name: 'product1',
+        reviews: {
+            Nicky: 'jkjkjlj',
+            Jany: 'jkjljlkjjkjjkjkj'
+        }
+    }
+];
+
+const users = [
+    {
+        name: 'Vasya'
+    },
+    {
+        name: "Petya"
+    }
+];
+
+router.get('/api/products', function(req, res) {
+    res.json(products);
+});
+
+router.get('/api/products/:id', function(req, res) {
+    res.send(products[req.params.id]);
+});
+
+router.get('/api/products/:id/reviews', function(req, res) {
+    res.send(products[req.params.id].reviews);
+});
+
+router.post('/api/products', function(req, res) {
```

Comment:
You didn't test it, did you? It wouldn't work without `body-parser` package

Reply:
Of course I've tested it. You can see express.json() in code below and this is an inbuild express equivalent of body-parser package

Reply:
Great! :)